models/docs.md

```diff
@@ -35,15 +35,15 @@ Name of campaign.
 {% enddocs %}
 
 {% docs attribution_type %}
-Attribution type: "CLICK_THROUGH_CONVERSION", "VIEW_THROUGH_CONVERSION", or "ALL_CONVERSION"
+Attribution type: "CLICK_THROUGH_CONVERSION", "VIEW_THROUGH_CONVERSION", or "ALL_CONVERSION" 
```

Comment:
For all of the edited descriptions in this file, can you add a bit more detail to what they are? For example, it is great to know what kind of timestamp the `created_at` field is. However, it would be great to add context around what the created_at field means.

Reply:
updated, but let me know if there's any remaining fields that could be made clearer